### PR TITLE
Contact Form

### PR DESCRIPTION
**Pull request by donice**
In the pull request, I:
- Implemented the contact form
- Styled the form
- Intergrated the Form Spree service to make a post request

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -221,7 +221,27 @@ <h4>Skills</h4>
...
+      <form action="https://formspree.io/f/mqkovwwj" method="POST">
+        <div class="form">
+          <label for="name">Your name</label><br>
+          <input id="name" type="text" name="name" placeholder="Donice Ubaru" maxlength="30" required>
+          <label for="email">Your email</label><br>
+          <input id="email" type="email" name="email" placeholder="doniceubaru1@gmail.com" required>
+          <label for="message">Your mesaage</label><br>
+          <textarea id="message" type="text" name="message" rows="4" cols="50" placeholder="Enter yout new message here"
+            maxlength="500" required></textarea>
+        </div>
```
Comment (Gambit142) [inline index.html]: - [ ] Please make the width of the `textarea` element to be the same as that of the other two `input` elements. This way, your design will be identical to that of the Figma design that you chose. 👍 💻 

| Your Design  | Figma Template |
| ------------- | ------------- |
| Screenshot_2023-06-14 154453 | image  |

File: index.html [hunk 2]
```diff
@@ -221,7 +221,27 @@ <h4>Skills</h4>
...
 
+        <button type="submit" id="submit-button">Start Collaboration</button>
+      </form>
```
Comment (Gambit142) [inline index.html]: - [ ] Please adjust the width of the form button to look identical to that of the Figma design. The current width is shorter than that of the Figma design. 💻 

| Your Design  | Figma Template |
| ------------- | ------------- |
| Screenshot_2023-06-14 155104 | image  |

#### Hint: It should be the same width as the input elements. 😎

File: index.html [hunk 3]
```diff
@@ -221,7 +221,27 @@ <h4>Skills</h4>
...
+    </div>
+  </section>
   <footer>
     <a href="/">Get my Resume<span class="material-symbols-outlined">
         open_in_new
```
Comment (Gambit142) [inline index.html]: - [ ] Based on the Figma design for your template, the footer should occupy the bottom part of the page. Your footer design has white space at the bottom of the page. Kindly adjust this. 👍 😎 

| Your Design  | Figma Template |
| ------------- | ------------- |
|
Screenshot_2023-06-14 160015 | image  |

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @donice @HossainAraf , 

Good job so far!

## Highlights
- Use of required HTML validation ✔️ 
- Set up Formspree on the for correctly ✔️
- Use of semantic HTML ✔️   

You still need to work on some issues to go to the next project, but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: style.css [hunk 4]
```diff
@@ -192,7 +197,7 @@ label {
 
 input,
 textarea {
-  width: 90%;
+  width: 100%;
```
Comment (Owusu-Desmond) [inline style.css]: - [ ] As requested by the previous code reviewer, the width of the textarea field is to be the same as that of the other two input elements. This way, your design will be identical to that of the Figma design that you chose. :point_down: 

  ### Your design
  [Screencast from 06-14-2023 08:14:11 PM.webm](https://github.com/donice/Portfolio-Microverse/assets/92303891/9432a20b-e7f1-43af-bafa-bfb1caddb08e)
  
  ### Figma design
  image

- [ ] While you were working on the requested changes, your project is having horizontal overflow on mobile view so make sure that you remove this overflow, as you can see in the video above to make your project responsive :+1:

File: index.html [hunk 5]
```diff
@@ -221,7 +221,28 @@ <h4>Skills</h4>
...
+        </div>
 
+        <button type="submit" id="submit-button">Start Collaboration</button>
+      </form>
+    </div>
+  </section>
+  
   <footer>
     <a href="/">Get my Resume<span class="material-symbols-outlined">
         open_in_new
```
Comment (Owusu-Desmond) [inline index.html]: - [ ] As requested by the previous reviewer, Based on the Figma design for your template, the footer should occupy the bottom part of the page. Your footer design has white space at the bottom of the page. Kindly adjust this. :+1:

  ### Your design
  image
  
  ### Figma design
  image

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @donice , :wave: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some changes you still need to work on before you move on to the next project but you are almost there.



 ## Highlights :+1: 
:heavy_check_mark: Good commit history.
:heavy_check_mark: Your project is well documented


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ofarouq310 — COMMENTED** [review]
Hi @donice, ✋  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

giphy

## 🔴 Invalid Code Review Request

- You have used the wrong project on the dashboard to review your contact form.

- Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by donice** [comment]
Hello @Ofarouq310 
I really do not understand the details of your reviews

**Comment by Ofarouq310** [comment]
@donice 
- Kindly note that you have submitted the wrong project to be reviewed.
On your dashboard, you have used your "Portfolio: Desktop version" instead of "Portfolio: Contact form". 
Hope that clarifies.

**Review by nahnahdev — APPROVED** [review]
## Project Status: APPROVED :green_circle: 
Hi @donice , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights :fire: 
- All requested changes implemented :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**